### PR TITLE
opendrop: init at 0.13.0

### PR DESCRIPTION
**Pull request by WolfangAukang**
###### Motivation for this change

#146777

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/fleep/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  pname = "fleep";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-yPYrJY7lNk1/bB7R8/J46ZAg/D8KYKJK0eEIRuMdEEw=";
+  };
+
+  pythonImportsCheck = [ "fleep" ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/fleep/default.nix:16]: The package has tests we can run.

Comment (WolfangAukang) [reply pkgs/development/python-modules/fleep/default.nix:29]: Yes, the Pypi package does not include them, so I had to fetch it from Github instead and patch the paths on the tests. Now it is verifying them.

**Comment by delneg** [comment]
Just FYI - https://github.com/delneg/opendrop-nix-service
I've published some code that I used to launch it on my home server, it works (although poorly) and I did transfer photos using airdrop successfully (using the code from this PR)
The code probably has problems as this is my first attempt to do a custom systemd service from scratch using nix.

**Comment by WolfangAukang** [comment]
@delneg That's good to know! I was not able to test this with my current hardware, so that gives a lot of confidence on the result of this package

**Pull request merged**